src/index.ts

```diff
@@ -55,44 +50,48 @@ function parseArguments() {
     console.log('Launching BiDi server.');
 
     const args = parseArguments();
-
     const bidiPort = args.port;
-    const browserExecutablePath = args.browser || process.env.BROWSER_PATH;
     const headless = args.headless !== 'false';
 
-    BidiServerRunner.run(
-      bidiPort,
-      (bidiServer: IServer) => {
-        return _onNewBidiConnectionOpen(
-          browserExecutablePath,
-          headless,
-          bidiServer
-        );
-      },
-      _onBidiConnectionClosed
-    );
+    BidiServerRunner.run(bidiPort, (bidiServer: IServer) => {
+      return _onNewBidiConnectionOpen(headless, bidiServer);
+    });
     console.log('BiDi server launched.');
   } catch (e) {
     console.log('Error', e);
   }
 })();
 
+/**
+ * On each new BiDi connection:
+ * 1. Launch Chromium (using Puppeteer for now).
+ * 2. Get `BiDi-CDP` mapper JS binaries using `mapperReader`.
+ * 3. Run `BiDi-CDP` mapper in launched browser.
+ * 4. Bind `BiDi-CDP` mapper to the `BiDi server`.
+ *
+ * @returns delegate to be called when the connection is closed
+ */
 async function _onNewBidiConnectionOpen(
-  browserExecutablePath: string,
   headless: boolean,
   bidiServer: IServer
-): Promise<BrowserProcess> {
-  // Launch browser.
-  const browserProcess = await launchBrowser(browserExecutablePath, headless);
-  // Get BiDi Mapper script.
+): Promise<() => void> {
+  // 1. Launch Chromium (using Puppeteer for now).
+  // Puppeteer should have downloaded Chromium during the installation.
+  // Use Puppetter's logic of launching browser as well.
+  const browser = await (puppeteer as any as PuppeteerNode).launch({
```

Comment:
For the record, my actual suggestion was to use [`puppeteer.createBrowserFetcher()`](https://github.com/puppeteer/puppeteer/blob/main/docs/api.md#puppeteercreatebrowserfetcheroptions). This patch is a little more indirect since it just uses whatever Chromium revision Puppeteer happens to bundle. If we need/want to control which exact revision to use for chromium-bidi in the future, we could still move towards using it.